flatland_server/src/world.cpp

```diff
@@ -97,12 +97,15 @@ World::~World() {
 }
 
 void World::Update(Timekeeper &timekeeper) {
```

Comment:
I think it might be cleaner here to create a SetPaused(bool) method on World and check World::paused_ rather than looking into InteractiveMarkerManager. I'm just wondering how we handle multiple things wishing to pause simulation (e.g. the pause/resume button idea, or some other system that wishes to pause sim). 

The actual pause logic here should work fine.